WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormDataModel.swift

```diff
@@ -55,6 +55,12 @@ protocol ProductFormDataModel {
     var soldIndividually: Bool? { get }
     // Whether stock status is available for the product.
     func isStockStatusEnabled() -> Bool
+
+    // Product downloads
+    var isDownloadable: Bool { get }
```

Comment:
I suggest to follow the format of the original data:
```suggestion
    var downloadable: Bool { get }
```

Reply:
Updated